libs/backend/src/intermediate-scripts/set-clock

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+usage () {
+    echo 'Usage: set-clock <iana-zone> <datetime-string>'
+    exit 1
+}
+
+if ! [[ "${#}" -eq 2 ]]; then
+    usage
+fi
+
+LOCALTIME_SYMLINK=`readlink /etc/localtime`
+TIMEZONE_DIRECTORY=/usr/share/zoneinfo
+
+# timedatectl sets timezone by symlinking /etc/localtime to a file in /usr/share/zoneinfo, but
+# because /etc is read-only on locked-down images, the setup-machine.sh script sets up a level
+# of indirection:
+#
+# /etc/localtime (read-only symlink) -->
+# /vx/config/localtime (writable symlink) -->
+# /usr/share/zoneinfo/...
+#
+# We account for that indirection, if relevant, here.
+#
+if [[ ! "${LOCALTIME_SYMLINK}" =~ "${TIMEZONE_DIRECTORY}".* ]]; then
+    ln -sf "${TIMEZONE_DIRECTORY}/${1}" "${LOCALTIME_SYMLINK}"
```

Comment:
This bit of logic (translated over from [vxsuite-complete-system/config/admin-functions/timedatectl](https://github.com/votingworks/vxsuite-complete-system/blob/14878282277ee2782b3379e10b39c067ad84d235/config/admin-functions/timedatectl)) was also lost in the transition from `kiosk-browser` to `libs/backend`.

The lack of this logic hasn't crashed our non-locked-down images, but I'm 99% sure that it will crash a locked-down image, so I'm getting ahead of it now.